.github/renovate.json5

```diff
@@ -98,7 +98,6 @@
     },
     {
       groupName: 'linter dependencies',
-      extends: ['group:linters'],
```

Comment:
if i understand correctly, it does not like groups being extended. any value in extending `packages:linters`?